crates/bevy_ecs/src/system/system_param.rs

```diff
@@ -1154,10 +1154,31 @@ impl<'w, 's> SystemParamFetch<'w, 's> for BundlesState {
     }
 }
 
+/// A [`SystemParam`] that reads the previous and current change ticks of the system.
+///
+/// A system's change ticks are updated each time it runs:
+/// - `last_change_tick` copies the previous value of `change_tick`
+/// - `change_tick` copies the current value of [`World::read_change_tick`]
```

Comment:
As you're making those fields private, it may be a little confusing to read in docs.rs where they will be hidden

Reply:
So, my thought was that users should be able to piece together what's going on because we expose methods of the same name.